recipes/kneed/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "kneed" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 39389c49b0cd9d976a3aaa8400aa9313bfb580818bc1ef60e0fcf4c9bb4f871e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
```

Comment:
Does this package need a compiler as it looks like it's pure Python? If it doesn't, the `build` section should be removed.